<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TweakPage</class>
 <widget class="QWidget" name="TweakPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>TweakPage</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>661</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbIp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Фильтр IP</string>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QListWidget" name="ipListWidget"/>
    <widget class="QLineEdit" name="lineEdit">
     <property name="inputMask">
      <string>000.000.000.000;_</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QCommandLinkButton" name="btnDeleteIp">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnUpdateIp">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Обновить/Добавить </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbConfig">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>251</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Конфигурация</string>
   </property>
   <widget class="QCheckBox" name="cbIpFilter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Фильтрация IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpecByte">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Значение байта &lt;/p&gt;&lt;p&gt;замены:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbSpecByte">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbSpecIface">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpecIface">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Номер специального &lt;/p&gt;&lt;p&gt;интерфейса:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="btnUpdateConfig">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkMode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Рабочий режим:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbWorkMode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <zorder>labelSpecIface</zorder>
   <zorder>cbIpFilter</zorder>
   <zorder>labelSpecByte</zorder>
   <zorder>sbSpecByte</zorder>
   <zorder>sbSpecIface</zorder>
   <zorder>btnUpdateConfig</zorder>
   <zorder>labelWorkMode</zorder>
   <zorder>cbWorkMode</zorder>
  </widget>
  <zorder>gbConfig</zorder>
  <zorder>gbIp</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
